src/EventStore.Core.Tests/TransactionLog/when_writing_prepare_record_to_file.cs

```diff
@@ -77,7 +77,7 @@ public class when_writing_prepare_record_to_file<TLogFormat, TStreamId> : Specif
 				Assert.AreEqual(p.EventId, _eventId);
 				Assert.AreEqual(p.EventStreamId, streamId);
 				Assert.AreEqual(p.ExpectedVersion, 1234);
-				Assert.AreEqual(p.TimeStamp, new DateTime(2012, 12, 21));
+				Assert.That(p.TimeStamp, Is.EqualTo(new DateTime(2012, 12, 21)).Within(20).Milliseconds);
```

Comment:
Can we use a tighter bound of 7 milliseconds here (or maybe 10)? (16 bits = 65536 ticks ~= 6.5 ms < 7 ms)

Reply:
sure, i'll set it to 7

Reply:
thanks!